Standards/scs-0217-v1-cluster-hardening.md

```diff
@@ -0,0 +1,475 @@
+
+Kubernetes defines the [*Pod security standards*][pod-security-standards]
+in the form of three policies that try to cover the range of the security spectrum.
+These policies can be found in the following list and define a list of restricted fields that can only be
+changed to a set of allowed values. An up-to-date list of these values can be found [here][pod-security-standards].
+
+- *Privileged*
+
+  Unrestricted policy, providing the widest possible level of permissions.
+  This policy allows for known privilege escalations.
+
+- *Baseline*
+
+  Minimally restrictive policy which prevents known privilege escalations.
+  Allows the default (minimally specified) Pod configuration.
+
+- *Restricted*
+
+  Heavily restricted policy, following current Pod hardening best practices.
+
+Kubernetes also offers the *Pod security* admission controller, which enforces
+the *Pod security standards* on a namespace level during pod creation.
+The admission controller defines the standard to be used with the three levels
+`privileged`, `baseline` and `restricted`. Each namespace can be configured to enforce
+a different control mode, which defines what action the control plane takes
+after a violation of the selected *Pod security* is detected.
+
+- `enforce`
+
+  Policy violations will cause the pod to be rejected.
+
+- `audit`
+
+  Policy violations will trigger the addition of an audit annotation to the event
+  recorded in the audit log, but are otherwise allowed.
+
+- `warn`
+
+  Policy violations will trigger a user-facing warning, but are otherwise allowed.
+
+Be aware, that `enforce` is not applied to workload resources, only to the pods created from their template.
+
+### Further measurements
+
+While researching this topic, further measurements were considered such as container image verification,
+distroless images, usage of `ImagePolicyWebhook`, network policy enforcement,
+container sandboxing and prevention of kernel module loading.
+Most of these were taken out of the document during writing due to either being the responsibility
+of the clusters' user (and therefore not possible to implement for the provider), being more relevant
+for high security clusters or changing the expected cluster environment too much, so that normally
+expected operations could potentially not work in such a modified cluster.
+These measurements will possibly be introduced in a future document about higher security clusters.
+
+## Standard
+
+This standard provides the baseline security requirements for a cluster in the SCS context. It tries
+to increase the security in contrast to a non-configured setup by requiring and recommending specific
+settings that provide different security improvements without being too intrusive.
+
+Kubernetes clusters MUST be updated regularly in order to receive bugfixes and security patches.
+For more information refer to the [SCS K8s Version Policy](scs-0210-v2-k8s-version-policy.md),
+which outlines the update policies of the SCS.
```

Comment:
```suggestion
which outlines the updated policies of the SCS.
```

Reply:
Updated in a2adca5b18c82097388b59a88dc665c0d2b72ab4